gensrc/thrift/Exprs.thrift

```diff
@@ -188,6 +193,129 @@ struct TFunctionCallExpr {
   2: optional i32 vararg_start_idx
 }
 
+struct TSlotDescriptor {
```

Comment:
Does changing/moving thrift definitions cause compatibility issues?

Reply:
I move some class definition from Descriptors.thrift to Expr.thrift avoid circular dependency, and all class have same namespace, I think it doesn't have side effect in code logic.
But it does cause unclear boundary, what do you think. @decster